client/packages/common/src/ui/components/inputs/TextInput/BasicTextInput.tsx

```diff
@@ -7,6 +7,7 @@ import {
 } from '@mui/material';
 
 export type BasicTextInputProps = StandardTextFieldProps & {
+  inputReference?: React.RefObject<HTMLInputElement>;
```

Comment:
I dunno about you, but I associate the term "ref" 100% with React refs, whereas the term "reference" can be much more broadly interpreted. Would it work to keep the prop name as `inputRef` so it's clearer for consumers n(and follows typical convention), and just rename it internally to avoid the name clash with the local `inputRef`?

Bit nitpicky, just a thought.

(Also, I just did a quick search and I couldn't find anywhere that this inputReference prop is actually being set -- am I missing one?)

Reply:
I suggest that you try your suggestion 😉 

You'll end up with problems in other components, as the type of the `inputRef` is incorrect. Giving the non-standard prop the non-standard name is the simple way to resolve the issue.

that was an initial attempt, and was later removed in the usage - so I've just taken it out of here now. still seems to work..